app/src/main/res/layout/timetable_lesson.xml

```diff
@@ -100,14 +101,14 @@
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_gravity="center_vertical"
+                    android:layout_marginTop="4dp"
                     android:fontFamily="sans-serif-condensed-light"
                     android:includeFontPadding="false"
-                    android:layout_marginBottom="-4dp"
```

Comment:
Why are you removing this margin?

Reply:
Look three lines up.

Reply:
The thing is, it's generally weird to put negative margins. So whatever the reason for me to put it here was, it must have been important. I remember I worked on this layout for days until I got it right (probably because the annotation that may be visible). I suspect this config is the right one for all variations of the lesson card, so I would leave it as it was.